### PR TITLE
build: add an action for handling feature requests

### PR DESCRIPTION
**Pull request by josephperrott**
Add feature request voting process github actions to the repo.

### CODE REVIEW

File: .github/workflows/feature-requests.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Feature request triage bot
+
+on:
+  schedule:
+    # Run at 13:00 every day
+    - cron: '0 13 * * *'
+
+jobs:
+  feature_triage:
+    if: github.repository == 'angular/angular-cli'
```
Comment (dgp1130) [inline .github/workflows/feature-requests.yml:10]: Is the motivation for this to disable the workflow for forks? If so, we should probably put a comment calling that out as it isn't very obvious. Someone would reasonably thinking that this is always `true` and not consider the forking use case.

File: .github/workflows/feature-requests.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: angular/dev-infra/github-actions/feature-request@354871956801929457d9f08482fb81f7012ed7c8
```
Comment (dgp1130) [inline .github/workflows/feature-requests.yml:13]: Does this require manual version bumps whenever the bot changes? I'm wondering if there's a way to more easily depend on the latest version or at least a release tag?

**Review by dgp1130 — APPROVED** [review]

File: .github/workflows/feature-requests.yml (hunk 2 above)
Comment (josephperrott) [reply .github/workflows/feature-requests.yml:13]: This is automatically updated by renovate.

File: .github/workflows/feature-requests.yml (hunk 1 above)
Comment (josephperrott) [reply .github/workflows/feature-requests.yml:10]: Added.

File: .github/workflows/feature-requests.yml (hunk 2 above)
Comment (dgp1130) [reply .github/workflows/feature-requests.yml:15]: Gotcha, if Renovate handles it, then that should be good enough.

Comment (josephperrott) [reply .github/workflows/feature-requests.yml:15]: It is the process used for keeping the other dev-infra based actions up to date, so we have already proved its success

**Pull request merged**